src/modules/trade/actions/place-trade.js

```diff
@@ -17,12 +17,16 @@ export const placeTrade = (marketId, outcomeId, tradeInProgress, doNotCreateOrde
     return dispatch(clearTradeInProgress(marketId))
   }
   const bnAllowance = createBigNumber(loginAccount.allowance)
+  const sharesDepleted = createBigNumber(tradeInProgress.sharesDepleted, 10)
+  const otherSharesDepleted = createBigNumber(tradeInProgress.otherSharesDepleted, 10)
+  const sharesProvided = sharesDepleted.eq(0) ? otherSharesDepleted.toString() : sharesDepleted.toString()
```

Comment:
Does `toString` always do what we want here?  Normally we used `toFixed` for base-10 string conversion.  IIRC `toString` can use scientific notation for large values and other weirdness that we don't want here.